CoreFoundation/String.subproj/CFString.h

```diff
@@ -152,14 +152,16 @@ since it is the default choice with Mac OS X developer tools.
 #endif
 
 #if DEPLOYMENT_RUNTIME_SWIFT
+    #if TARGET_OS_MAC
+        #define _CF_CONSTANT_STRING_SWIFT_CLASS $s15SwiftFoundation19_NSCFConstantStringCN
+    #else
+        #define _CF_CONSTANT_STRING_SWIFT_CLASS $s10Foundation19_NSCFConstantStringCN
+    #endif
```

Comment:
Do we care about the different versions of the ABI?

Reply:
No, because Foundation is in lockstep with Swift anyway.

Reply:
SGTM